releasenotes/notes/allow-empty-host-from-dogstatsd-7895ace81adbc974.yaml

```diff
@@ -0,0 +1,6 @@
+---
+fixes:
+  - |
+    Allow metric to have an empty host when coming from dogstatsd. This allows
```

Comment:
I'd say `Take into account empty hosts on metrics coming from dogstatsd, instead of ignoring them and applying the Agent's hostname.`